chef/cookbooks/nfs-server/recipes/default.rb

```diff
@@ -55,7 +55,6 @@
 service rpc_service do
   action [:enable, :start]
 end
-utils_systemd_service_restart rpc_service
```

Comment:
I am probably misunderstanding something, but on a fresh SP2 install the service script for `rpcbind` is `notify` and not `oneshot`. And for those it's ok to set `Restart` other than no.

Reply:
rpc is iirc autostarted by nfs, so removing that should be fine.